packages/server/src/api/muster/muster.postgres.controller.it.ts

```diff
@@ -0,0 +1,81 @@
+import { expect } from 'chai';
+import { expectNoErrors } from '../../util/test-utils/expect';
+import { seedAll } from '../../util/test-utils/seed';
+import { TestRequest } from '../../util/test-utils/test-request';
+import { User } from '../user/user.model';
+import { Unit } from '../unit/unit.model';
+
+
+describe('Muster Controller Compliance', () => {
+  const basePath = '/api/muster';
+  let req: TestRequest;
+  let seedData: any[];
+  let orgId: number;
+  let unitId: number;
+
+  beforeEach(async () => {
+    seedData = await seedAll();
+    req = new TestRequest(basePath);
+    req.setUser(await User.findOne());
+
+    orgId = seedData[0].org.id;
+    unitId = seedData[0].units.filter(((u: Unit) => u.name === 'Unit 1' && u.org!.id === orgId)).map((unit: { id: any }) => unit.id)[0];
```

Comment:
why `unit: {id: any}` in the map function? Why not just `unit: Unit` or at the very least make `id: number` ?

Reply:
Also using 'find' might be more appropriate than `map` so you return single value and don't need to index an array.

Reply:
simplified

Reply:
good catch!